packages/core/review-workflows/server/src/services/metrics/weekly-metrics.ts

```diff
@@ -60,7 +60,7 @@ export default ({ strapi }: { strapi: Core.Strapi }) => {
       let weeklySchedule = currentSchedule;
 
       if (!currentSchedule || !lastWeeklyUpdate || lastWeeklyUpdate + ONE_WEEK < now.getTime()) {
-        weeklySchedule = getWeeklyCronScheduleAt(add(now, { seconds: 10 }));
+        weeklySchedule = getWeeklyCronScheduleAt(add(now, { seconds: 15 }));
```

Comment:
why did you change also this value? just for curiosity

Reply:
to keep it consistent with the upload metrics, 10 seconds seemed to little